tests/integration-tests/requirements.txt

```diff
@@ -17,3 +17,19 @@ PyYAML
 retrying
 troposphere
 untangle
+marshmallow
+aws-cdk.core
+aws-cdk.aws-batch
+aws-cdk.aws-codebuild
+aws-cdk.aws-dynamodb
+aws-cdk.aws-ec2
+aws-cdk.aws-efs
+aws-cdk.aws-events
+aws-cdk.aws-fsx
+aws-cdk.aws-imagebuilder
+aws-cdk.aws-iam
+aws-cdk.aws-logs
+aws-cdk.aws-route53
+aws-cdk.aws-ssm
+aws-cdk.aws-sqs
+aws_cdk.aws_lambda
```

Comment:
minor: aws_cdk.aws_lambda here should be aws_cdk.aws-lambda. 
minor2: let's put the imports in alphabetical order, in this way it's simpler to find them.